### PR TITLE
[FLOSS H2] Reinstate Mounting tutorial

### PR DESCRIPTION
**Pull request by Bujuhu**
## Basics

- [x] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [x] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation (see [Documentation Guidelines](https://www.libelektra.org/devgettingstarted/documentation))
- [ ] I fixed all affected decisions (see [Decision Process](https://www.libelektra.org/decisions/decision-process))
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://www.libelektra.org/devgettingstarted/coding))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you.

### CODE REVIEW

**Review by Dynamichost96 — COMMENTED** [review]
# Review

### Documentation is introductory, concise, good to read and describes everything what the PR does
The documentation fulfills the criteria the changed sentences are easier to read and understand, in comparison to the prior.

### Examples are well chosen and understandable
No new Examples where provided. 

### Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
No code was added.

### APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
No code was added.

### Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)
No code was added.

File: doc/tutorials/mount.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-<!-- TODO [new_backend]: Outdated, only kept as reference until an updated version is written
-Elektra provides a global key database, that can integrate configuration in various formats.
+In ths tutorial we will go over mounting configuration files to enable non-elektrified applications to be configured through Elektra.
 
-Conversely configuration managed by Elektra can be integrated into applications.
 The best way of integrating Elektra into applications is to [elektrify](/doc/help/elektra-glossary.md) them.
 
-A simpler form of integration is to let Elektra directly use configuration files as they are present on the system.
-Thus applications can read the configuration files and changes in the key database will be picked up by applications.
+To use elketra with non-elektrified applications we can also let Elektra sync configuration files on the file system and the elektra key-storage.
```
Comment (stefnotch) [inline doc/tutorials/mount.md:7]: Typo here, `elketra` should be `Elektra`

**Review by stefnotch — CHANGES_REQUESTED** [review]

File: doc/tutorials/mount.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-<!-- TODO [new_backend]: Outdated, only kept as reference until an updated version is written
-Elektra provides a global key database, that can integrate configuration in various formats.
+In ths tutorial we will go over mounting configuration files to enable non-elektrified applications to be configured through Elektra.
 
-Conversely configuration managed by Elektra can be integrated into applications.
 The best way of integrating Elektra into applications is to [elektrify](/doc/help/elektra-glossary.md) them.
 
-A simpler form of integration is to let Elektra directly use configuration files as they are present on the system.
-Thus applications can read the configuration files and changes in the key database will be picked up by applications.
+To use Elketra with non-elektrified applications we can also let Elektra sync configuration files on the file system and the Elektra key-storage.
```
Comment (markus2330) [inline doc/tutorials/mount.md:7]:
```suggestion
To use Elektra with non-elektrified applications we can also let Elektra sync configuration files on the file system and  Elektra's key-value storage.
```

**Review by markus2330 — CHANGES_REQUESTED** [review]
Great job, please fix the spelling mistakes and add them to scripts/sed.

**Comment by markus2330** [comment]
jenkins build libelektra please

**Comment by markus2330** [comment]
Great job!

**Pull request merged**